src/customize-pop/keyboard-settings.md

```diff
@@ -0,0 +1,64 @@
+# Keyboard Settings
+
+Option to set input Sources, input source switching, special character entry options, and keyboard shortcuts are located in `Settings` ➞ `Keyboard`.
+
+![Keyboard Settings](/images/keyboard-settings/keyboard-settings.png)
+
+| Option | Function |
+|--------|----------|
+| Input Sources | Choose multiple input languages and layouts for a single keyboard. Sources can be moved up or down to determine priority. |
+| Input Source Switching | Set a keyboard shortcut for switching between your chosen sources. |
+| Special Character Key | Designate keys that enable entry of special characters. |
+| Keyboard Shortcuts | View, edit, and add keyboard shortcuts. |
+
+## Adding an Input Source
+
+1. Click the `+` to display available input sources. Select an input source from the list, or click the three vertical dots to perform a keyword search.
+
+    ![Display Sources](/images/keyboard-settings/display-sources.png)
+
+2. Select a language from the list.
+
+    ![Select Source](/images/keyboard-settings/select-source.png)
+
+3. Select a keyboard layout and click `Add`.
+
+> **Keyboard Layout vs. Keyboard Input Language**: A keyboard language refers to the available keys required to input the required characters and symbols for a given language. A keyboard layout is the physical organization of those keys.
+
+![Select Layout](/images/keyboard-settings/select-layout.png)
+
+Display keyboard layouts by clicking the three vertical dots on the right and clicking `View Keyboard Layout`
```

Comment:
We really should have a picture or video of showing the keyboard layout.

Video would be best because then you can show how the keys light up while you type. This is a good troubleshooting tool to show whether keystrokes are being registered/mapped correctly.

Somewhere in this section would be a good place to link to the Keyboard Configurator/Launch too maybe.

Reply:
I've added a webm to demonstrate the keyboard layout feature. At this point I've tried to avoid adding information exclusive to users of System76 hardware, since this documentation is intended for anyone that installs and uses Pop!_OS on any PC or VM. I'm not ruling it out at this point, but it's not a priority. I've added a comment in case this is added at a later date.